### DISCUSSION TITLE
where is reference to seach service in code?

### DISCUSSION THREAD

**Discussion by iamvishalpatel** [Azure-Samples/chatgpt-quickstart#103] [General]
in chat.py I only see reference to openai but not search. Thought this solution first retrive vector from Az Search and feed to Az OpenAI based on the architecture diagram

**Comment by pamelafox** [comment]
Hm, which architecture diagram are you referring to? It sounds like the one for https://github.com/Azure-Samples/azure-search-openai-demo/
Is the diagram incorrect for this one somewhere?

**Reply by iamvishalpatel** [reply]
![openaisearch](https://github.com/Azure-Samples/openai-chat-app-quickstart/assets/40643531/066b8c65-c8d6-4dd6-bf3c-fd719c8c5027)

isn't it using Azure search?

**Reply by pamelafox** [reply]
Ah, Azure Cognitive Services is where OpenAI comes from. I could use a single logo for that, perhaps that's confusing! I think you want this one:
https://github.com/Azure-Samples/azure-search-openai-demo/

**Reply by iamvishalpatel** [reply]
thanks - I was checking that one too but I do not see any reference of azure search in Chat.tsx

**Reply by pamelafox** [reply]
The Chat.tsx is the frontend file, it makes a call to the backend which then does a search call. There's an explanation of the code flow here: https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/docs/customization.md#customizing-the-backend
